integration-tests/actions/vrfv2_actions/vrfv2_steps.go

```diff
@@ -91,188 +110,194 @@ func CreateVRFV2Jobs(
-[EVM.KeySpecific.GasEstimator]
-PriceMax = '%d gwei'
-`
-	//todo - make evmKeySpecificConfigTemplate for multiple eth keys
-	evmKeySpecificConfig := fmt.Sprintf(evmKeySpecificConfigTemplate, nativeTokenPrimaryKeyAddress, vrfv2_constants.MaxGasPriceGWei)
-	tomlConfigWithUpdates := fmt.Sprintf("%s\n%s", config.BaseVRFV2NetworkDetailTomlConfig, evmKeySpecificConfig)
-
-	//todo - this does not show up??
-	newEnvLabel := "updatedWithRollout=true"
-	testEnvironmentAfterRedeployment := SetupVRFV2Environment(
-		t,
-		testNetwork,
-		tomlConfigWithUpdates,
-		existingTestEnvironment.Cfg.Namespace,
-		newEnvNamespacePrefix,
-		newEnvLabel,
-		newEnvTTL,
-	)
-
-	err = testEnvironmentAfterRedeployment.RolloutStatefulSets()
-	require.NoError(t, err, "Error performing rollout restart for test environment")
-
-	err = testEnvironmentAfterRedeployment.Run()
-	require.NoError(t, err, "Error running test environment")
-
-	//need to get node's urls again since port changed after redeployment
-	chainlinkNodesAfterRedeployment, err := client.ConnectChainlinkNodes(testEnvironmentAfterRedeployment)
-	require.NoError(t, err)
-
-	return vrfV2Contracts, chainlinkNodesAfterRedeployment, vrfV2jobs, testEnvironmentAfterRedeployment
-}
-
-func SetupVRFV2Environment(
-	t *testing.T,
-	testNetwork blockchain.EVMNetwork,
-	networkDetailTomlConfig string,
-	existingNamespace string,
-	namespacePrefix string,
-	newEnvLabel string,
-	ttl time.Duration,
-) (testEnvironment *environment.Environment) {
-	gethChartConfig := getGethChartConfig(testNetwork)
-
-	if existingNamespace != "" {
-		testEnvironment = environment.New(&environment.Config{
-			Namespace: existingNamespace,
-			Test:      t,
-			TTL:       ttl,
-			Labels:    []string{newEnvLabel},
-		})
-	} else {
-		testEnvironment = environment.New(&environment.Config{
-			NamespacePrefix: fmt.Sprintf("%s-%s", namespacePrefix, strings.ReplaceAll(strings.ToLower(testNetwork.Name), " ", "-")),
-			Test:            t,
-			TTL:             ttl,
-		})
+	if err != nil {
+		return errors.Wrap(err, ErrSendingLinkToken)
 	}
-
-	cd, err := chainlink.NewDeployment(1, map[string]any{
-		"toml": client.AddNetworkDetailedConfig("", networkDetailTomlConfig, testNetwork),
-		//need to restart the node with updated eth key config
-		"db": map[string]interface{}{
-			"stateful": "true",
-		},
-	})
-	require.NoError(t, err, "Error creating chainlink deployment")
-	testEnvironment = testEnvironment.
-		AddHelm(gethChartConfig).
-		AddHelmCharts(cd)
-	err = testEnvironment.Run()
-	require.NoError(t, err, "Error running test environment")
-	return testEnvironment
+	return chainClient.WaitForEvents()
 }
 
-func getGethChartConfig(testNetwork blockchain.EVMNetwork) environment.ConnectedChart {
-	evmConfig := eth.New(nil)
-	if !testNetwork.Simulated {
-		evmConfig = eth.New(&eth.Props{
-			NetworkName: testNetwork.Name,
-			Simulated:   testNetwork.Simulated,
-			WsURLs:      testNetwork.URLs,
-		})
-	}
-	return evmConfig
-}
+//func SetupVRFV2Universe(
```

Comment:
Is this large block of commented code still needed? If we aren't sure I am fine with leaving it for now.